gridcoinresearch.pro

```diff
@@ -42,17 +42,6 @@ OBJECTS_DIR = build
 MOC_DIR = build
 UI_DIR = build
 
-# use: qmake "RELEASE=1"
-contains(RELEASE, 1) {
-    # Mac: compile for maximum compatibility (10.5, 32-bit)
-    macx:QMAKE_CXXFLAGS += -mmacosx-version-min=10.5 -arch x86_64 -isysroot /Developer/SDKs/MacOSX10.5.sdk
-
-    !windows:!macx {
-        # Linux: static link
-        LIBS += -Wl,-Bstatic
```

Comment:
@TheCharlatan This will not affect Linux builds as we'll use depends, right?

Reply:
Yeah but the conditions excluded Windows and OSX, so it was only used on Linux.

Reply:
is this why i cannot build 32bit windwos exe lol

Reply:
Should I close this pull request then?